### DISCUSSION TITLE
Fetch data with Foreign table but using group by

### DISCUSSION THREAD

**Discussion by BrunoFerreira95** [supabase/supabase#14125] [Feature Requests]
Introduction:

I was working here in Service Security with a project of Store, when I have to receive products and send products, when I was being using nextjs and supabase.

Problem:

Before I send products I need to fetch all entries and make a group by for the goods id, to make a sum of quantities of products.

I using the version of JavaScript 2. x, and when I ask for help on the supabase AI, they suggest a lot of possibility but they are not implemented.

Suggest or Feat:

The group by its an import SQL method when the purpose the team could implement for the next versions.

Solution: 

I will need to fetch the data and make the group by in JavaScript language using programming.

Conclusion: 

I’m a junior developer in Brazil company, and I am trying to help the community with this post, so be patient about maybe it’s already implemented but I do not find the solution.

**Comment by omarelshiltawi** [comment]
let me know if this helps https://chat.elapse.ai/conversation/19dd64f0-c1ee-489f-bff6-3b7467b7b243

**Reply by BrunoFerreira95** [reply]
This is a great help. Thanks for all
